src/main/java/net/minecraftforge/event/level/ChunkEvent.java

```diff
@@ -58,9 +58,30 @@ public ChunkAccess getChunk()
      **/
     public static class Load extends ChunkEvent
     {
+        private final boolean newChunk;
+
+        @Deprecated
```

Comment:
This should be marked as terminally deprecated since 1.19.3, as well as being internal API.
```suggestion
        @Deprecated(forRemoval = true, since = "1.19.3")
        @ApiStatus.Internal
```